tests/functional/basic/test_jaffle_shop.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from dbt.tests.util import run_dbt, get_manifest
+
+
+from tests.fixtures.jaffle_shop import models, seeds, project_config_update
```

Comment:
You mean the 'models', 'seeds' and 'project_config_update'?  They are used. They override the base fixtures that come from the fixtures/project.py file.

Reply:
Flake gives shows the following warning:

> F401: 'tests.fixtures.jaffle_shop.seeds' imported but unused

I guess an `isort` or `black` would remove this line. 

Pytest recommends to [overwrite fixtures](https://docs.pytest.org/en/7.0.x/how-to/fixtures.html#overriding-fixtures-on-various-levels) by using `conftest`, within a test module or with direct test parameterization. By importing the fixtures it is _kinda_ like defining a fixture on test module level, but not according to the pytest overriding fixture way. And, flake does not like it.

Reply:
Yeah, this isn't ideal from flake and mypy considerations. But the whole point here is to package up standard projects and use them in a single test. Putting them in a conftest.py defeats that because it would apply to all tests in the directory. If you can come up with a better way to import the fixtures that we need for a single project, I'd love to hear it.